pkg/ddl/backfilling_operators.go

```diff
@@ -639,6 +640,25 @@ func (w *indexIngestWorker) HandleTask(rs IndexRecordChunk, send func(IndexWrite
 	send(result)
 }
 
+func (w *indexIngestWorker) initSessCtx() {
+	if w.se == nil {
+		sessCtx, err := w.sessPool.Get()
+		if err != nil {
+			w.ctx.onError(err)
+			return
+		}
+		w.restore = restoreSessCtx(sessCtx)
+		if err := initSessCtx(sessCtx,
+			w.reorgMeta.SQLMode,
+			w.reorgMeta.Location,
+			w.reorgMeta.ResourceGroupName); err != nil {
+			w.ctx.onError(err)
+			return
+		}
```

Comment:
Is it necessary to call restoreSessCtx since initSessCtx use the reorgMeta to init sessCtx.